tests/NRedisStack.Tests/NRedisStack.Tests.csproj

```diff
@@ -23,7 +23,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
     <PackageReference Include="NetTopologySuite" Version="2.5.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
-    <PackageReference Include="StackExchange.Redis" Version="2.6.122" />
+    <PackageReference Include="StackExchange.Redis" Version="2.7.4" />
```

Comment:
Can we get all of these versions into a single props file?

Reply:
@slorello89 do you know if its possible?

Reply:
@shacharPash just try it

Reply:
Yes, you can specify all of the dependant versions in a separate file

Reply:
File is called `Directory.Packages.props`, see https://devblogs.microsoft.com/nuget/introducing-central-package-management/

Reply:
did it 👍